DuckDuckGo/DuckPlayer/DuckPlayer.swift

```diff
@@ -184,7 +184,12 @@ final class DuckPlayer: DuckPlayerProtocol {
     }
     
     public func getUserValues(params: Any, message: WKScriptMessage) -> Encodable? {
-        encodeUserValues()
+        let duckPlayerExperiment = DuckPlayerLaunchExperiment()
+        if duckPlayerExperiment.isEnrolled && duckPlayerExperiment.isExperimentCohort {
+            return encodeUserValues()
+        }
+        return nil
```

Comment:
Maybe we could add a comment here explaining the different behavior when returning nil? Or maybe just the asana [task](https://app.asana.com/0/1204167627774280/1208509174931754/f) for context

Reply:
sure